src/charts/helpers/style.js

```diff
@@ -95,7 +95,9 @@ module.exports = (function() {
                     }
                 });
 
-                result = elem.outerHTML;
+                // fix for IE 11 not supporting svg.outerHTML
+                result = $('<div>').append($(elem).clone()).html();
```

Comment:
Mmmm... is this the jQuery '$'?

Reply:
If so, we should probably use another method.

Reply:
yes, sorry, I wanted to use something that wasn't jquery but this seemed like the shortest way of doing it.  I will look for another alternative, so at least it doesn't fail testing.

Reply:
Ok, this XMLserializer solution works instead of jquery.
https://stackoverflow.com/a/45120072